copulas/univariate/beta.py

```diff
@@ -36,7 +37,9 @@ def _get_model(self):
     def _fit_beta(self, X):
         """Fit the beta parameters to the data.
         """
-        self.a, self.b, self.loc, self.scale = beta.fit(X)
+        self.loc = np.min(X)
```

Comment:
To be consistent with truncated gaussian, we should estimate loc/scale directly.